src/main/resources/feedbackQuestionTextSubmissionFormTemplate.html

```diff
@@ -1,7 +1,9 @@
 <div class="panel panel-default panel-body"
-    ${disabled} id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    data-length-text-id="responseLength-${questionIndex}-${responseIndex}" data-recommended-text="${recommendedLength}">
+     data-is-session-closed="${isSessionClosed}"
+     id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
+     name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
+     data-length-text-id="responseLength-${questionIndex}-${responseIndex}"
+     data-recommended-text="${recommendedLength}">
```

Comment:
@damithc Advice on this? I would say that restricting the number of changes a PR makes to be strictly in keeping with the change being introduced in it is preferable.